docs/src/content/docs/ko/guides/css-and-tailwind.mdx

```diff
@@ -154,92 +168,90 @@ yarn create astro --template starlight/tailwind
 
     </Tabs>
 
-3.  `src/tailwind.css` 파일과 같은 Tailwind의 기본 스타일에 대한 CSS 파일을 만드세요.
+3. Astro가 스캐폴딩한 `src/styles/global.css` 파일의 콘텐츠를 Starlight와의 호환성을 위해 다음 코드로 교체합니다.
 
     ```css
-    /* src/tailwind.css */
-    @tailwind base;
-    @tailwind components;
-    @tailwind utilities;
+    /* src/styles/global.css */
+    @layer base, starlight, theme, components, utilities;
+
+    @import '@astrojs/starlight-tailwind';
+    @import 'tailwindcss/theme.css' layer(theme);
+    @import 'tailwindcss/utilities.css' layer(utilities);
     ```
 
-4.  Tailwind 기반 스타일을 사용하고 기본 스타일을 비활성화하려면 Astro 구성 파일을 업데이트하세요.
+4. Tailwind CSS 파일을 사용하도록 Starlight 구성을 업데이트합니다.
 
-    ```js {11-12,16-17}
+    ```js ins={11-12}
     // astro.config.mjs
     import { defineConfig } from 'astro/config';
     import starlight from '@astrojs/starlight';
-    import tailwind from '@astrojs/tailwind';
+    import tailwindcss from '@tailwindcss/vite';
 
     export default defineConfig({
     	integrations: [
     		starlight({
-    			title: 'Tailwind를 사용한 문서',
+    			title: 'Docs with Tailwind',
```

Comment:
```suggestion
    			title: 'Tailwind를 사용한 문서',
```

Do you think we can re-use the previous title or maybe that was intentional and it's better in English?